src/sdl_font.cpp

```diff
@@ -0,0 +1,677 @@
+#if defined(TILES)
+#include "sdl_font.h"
+#include "output.h"
+
```

Comment:
It looks like it is missing following code:

https://github.com/CleverRaven/Cataclysm-DDA/blob/8cfe0e71f8ca73bd44296a1c2fc40884322ea292/src/sdltiles.cpp#L67-L77

Reply:
Oh, thanks :)